de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/session/internal/ResourceActivityQueuer.java

```diff
@@ -0,0 +1,85 @@
+package de.fu_berlin.inf.dpp.session.internal;
+
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+
+import de.fu_berlin.inf.dpp.activities.IActivity;
+import de.fu_berlin.inf.dpp.activities.IResourceActivity;
+import de.fu_berlin.inf.dpp.activities.SPath;
+import de.fu_berlin.inf.dpp.session.IActivityQueuer;
+
+/**
+ * This Resource based Queue gets a {@code Set} of {@link SPath}s, which should
```

Comment:
Please mention that activities for the set of SPaths are queued. (Activities are never mentioned in this part of the javadoc.)